cmd/easi/test/test.go

```diff
@@ -21,6 +21,7 @@ func Server() {
 	c := exec.Command(
 		"go",
 		"test",
+		"-cover",
```

Comment:
```suggestion
		"-coverprofile=coverage.out",
```

Reply:
In addition to this, it'd be nice to get the total test coverage (`go tool cover` should generate that) and upload the test artifacts to CI.